src/binary/ibc-setup/commands/balances.ts

```diff
@@ -0,0 +1,86 @@
+import os from 'os';
+import path from 'path';
+
+import { stringToPath } from '@cosmjs/crypto';
+import { GasPrice } from '@cosmjs/launchpad';
+import { DirectSecp256k1HdWallet } from '@cosmjs/proto-signing';
+
+import { Coin } from '../../../codec/cosmos/base/v1beta1/coin';
+import { IbcClient } from '../../../lib/ibcclient';
+import { registryFile } from '../../constants';
+import { loadAndValidateApp } from '../utils/load-and-validate-app';
+import { loadAndValidateRegistry } from '../utils/load-and-validate-registry';
+import { resolveHomeOption } from '../utils/options/shared/resolve-home-option';
+import { resolveKeyFileOption } from '../utils/options/shared/resolve-key-file-option';
+import { resolveMnemonicOption } from '../utils/options/shared/resolve-mnemonic-option';
+
+import { Flags, getAddresses, Options } from './keys-list';
+
+export { Options }; // re-export for testing
```

Comment:
Why not just import it from the same place in tests?

Reply:
765bc7f

Reply:
I thought it's cleaner to import `Options` from `balances.ts` instead of `keys-list.ts`  in `balances.spec.ts`.


But it doesn't really matter, changed.